jupyter/jupyter-web-app/overlays/application/application.yaml

```diff
@@ -1,16 +1,11 @@
 apiVersion: app.k8s.io/v1beta1
 kind: Application
 metadata:
-  name: jupyter-web-app
+  name: $(generateName)
```

Comment:
Why are we parameterizing the name rather than just explicitly setting it here? What was wrong with just hardcoding the name here?